people/CMakeLists.txt

```diff
@@ -44,7 +44,10 @@ set(srcs
 
 set(LIB_NAME "pcl_${SUBSYS_NAME}")
 include_directories("${CMAKE_CURRENT_SOURCE_DIR}/include")
+
 PCL_ADD_LIBRARY(${LIB_NAME} COMPONENT ${SUBSYS_NAME} SOURCES ${srcs} ${incs} ${impl_incs})
+target_link_libraries(${LIB_NAME} pcl_common pcl_io pcl_kdtree pcl_segmentation pcl_search pcl_features pcl_sample_consensus pcl_filters pcl_visualization)
```

Comment:
How the compiler succeeded before this MR, without this?

And `pcl_io` depends on `pcl_common`, so usually it should'n be necessary to link explicitely against `pcl_io` (except the targets are badly defined).

Reply:
People don't use the VTKlibraries directly, but somehow exposes vtk types PCLVisualizer is included. But originally all vtk libraries was linked to I/O, which kinda saved ie. some tools, which then showed linking errors after I limited the IO linking to vtk libraries.
So dependencies of pcl_people survived on transient dependecies from IO.

Reply:
Sorry, I still don't see why `pcl_people` has to be linked to `pcl_io` and `pcl_features`?

Reply:
I just added those that was defined at line 3. But we can try remove some, if we think they aren't necessary.

Reply:
Removed io and features. Left the rest in, as they are at least some includes from them in the .hpp files.